src/main/java/org/assertj/core/util/diff/Delta.java

```diff
@@ -136,6 +136,7 @@ int lineNumber() {
   }
 
   String formatLines(List<T> lines) {
+    //TODO see what we should use here
     return STANDARD_REPRESENTATION.format(lines, DEFAULT_START, DEFAULT_END, ELEMENT_SEPARATOR_WITH_NEWLINE, "   ");
```

Comment:
Because the method is part of the `StandardRepresentation` API and not part of the `Representation`. If it is OK for you we can use `CONFIGURATION_PROVIDER.representation().toStringOf(lines)` then I can change that as well and deprecate the `StandardRepresentation.STANDARD_REPRESENTATION` and remove it in the version after this one.